frontend/src/components/interviews/helpers/get-interviews-columns.helper.ts

```diff
@@ -0,0 +1,41 @@
+import { InterviewsTableAccessor } from 'components/interviews/common/enums/enums';
+import { InterviewsTableRow } from 'components/interviews/common/types/types';
+import { Column } from 'react-table';
+
+import { CategoryCell } from '../components/category-cell/category-cell';
+import { DateCell } from '../components/date-cell/date-cell';
+import { StatusCell } from '../components/status-cell/status-cell';
+
+const getInterviewsColumns = (): Column<InterviewsTableRow>[] => {
+  return [
+    {
+      Header: 'Id',
+      accessor: InterviewsTableAccessor.ID,
```

Comment:
```suggestion
      Header: 'Id',
      accessor: InterviewsTableAccessor.ID,
      Cell: IdCell
```
should be a link to `/interviews/:id`